cmd/deploy/deploy.go

```diff
@@ -0,0 +1,58 @@
+package deploy
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/stormkit-io/stormkit-cli/api"
+	"github.com/stormkit-io/stormkit-cli/cmd"
+	"github.com/stormkit-io/stormkit-cli/model"
+	"github.com/stormkit-io/stormkit-cli/stormkit"
+)
+
+// deployCmd represents the deploy command
+var deployCmd = &cobra.Command{
+	Use:   "deploy",
+	Short: "A brief description of your command",
+	Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	RunE: runDeploy,
+}
+
+func init() {
+	cmd.GetRootCmd().AddCommand(deployCmd)
+
+	// Here you will define your flags and configuration settings.
```

Comment:
**question** Shall we remove these comments?

Reply:
Done [03816db](https://github.com/stormkit-io/stormkit-cli/pull/9/commits/03816dbef80ea4b7d23a884380ddf66741b3e2d8)